src/scripts/main.js

```diff
@@ -1,3 +1,56 @@
 'use strict';
 
-// write code here
+const MAX_SIZE = 10;
+const MIN_SIZE = 2;
+
+function refreshButtons(size, appendButton, removeButton) {
+  if (size >= MAX_SIZE) {
+    appendButton.disabled = true;
+  } else if (size <= MIN_SIZE) {
+    removeButton.disabled = true;
+  } else {
+    appendButton.disabled = false;
+    removeButton.disabled = false;
+  }
```

Comment:
Let's improve readability and make something like this
```suggestion
  if (size >= MAX_SIZE) {
    appendButton.disabled = true;
    return;
  } 
  
  if (size <= MIN_SIZE) {
    removeButton.disabled = true;
    return;
  } 
  
  
    appendButton.disabled = false;
    removeButton.disabled = false;
  }
```